Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -128,16 +133,20 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
         didSet {
             lockImageView.hidden = (url?.scheme != "https")
             let t = url?.absoluteString
+
             if t?.hasPrefix("http://") ?? false {
                 locationLabel.text = t!.substringFromIndex(advance(t!.startIndex, 7))
             } else if t?.hasPrefix("https://") ?? false {
                 locationLabel.text = t!.substringFromIndex(advance(t!.startIndex, 8))
             } else if t == "about:home" {
-                let placeholderText = NSLocalizedString("Search or enter address", comment: "The text shown in the URL bar on about:home")
-                locationLabel.attributedText = NSAttributedString(string: placeholderText, attributes: [NSForegroundColorAttributeName: UIColor.grayColor()])
+                // This will show the placeholder label instead of the url.
+                url = nil
+                locationLabel.attributedText = BrowserLocationView.PlaceholderText
+                return
```

Comment:
Oh you're right! Neat. I don't even think that's documented in the Swift docs. They have a note for setting things during init() but here its just "If you assign a value to a property within its own didSet observer, the new value that you assign will replace the one that was just set."